spec/android_spec.rb

```diff
@@ -132,7 +132,7 @@
         out = 'Failure [INSTALL_FAILED_ALREADY_EXISTS]'
 
         device = DeviceAPI::Android::Device.new(serial: 'SH34RW905290')
-        allow(Open3).to receive(:capture3) { [out, '', $STATUS_ZERO] }
+        allow(Open3).to receive(:capture3) { [out, '', STATUS_OK] }
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.